src/core/services/gesture/gesture.js

```diff
@@ -318,6 +331,19 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
       }
     });
 
+  function getTouchAction() {
+    var testEl = angular.element('<div>')[0];
```

Comment:
This one is the same as doing `document.createElement('div')`, without having to wrap it with jqLite.